src/schemas/index.js

```diff
@@ -0,0 +1,28 @@
+var apollo = require('apollo-server-express');
+var userSchema = require('./user');
+var itemSchema = require('./item');
+var itemInCartSchema = require('./item_in_cart');
+var cartSchema = require('./cart');
+
+const linkSchema = apollo.gql`
+
+  type Query {
+    _: Boolean
```

Comment:
You could test an empty object and see what happens. Another suggestion here is to add a kind of a `version` field. I did it in one project of mine instead of this `_: Boolean` nonsense workaround =P